service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -119,6 +131,34 @@ public void unadoptSite(JWTData userData, int siteId) {
         .execute();
   }
 
+  @Override
+  public void forceUnadoptSite(JWTData userData, int siteId) {
+    isAdminCheck(userData.getPrivilegeLevel());
+    checkSiteExists(siteId);
+    if (!isAlreadyAdopted(siteId)) {
+      throw new WrongAdoptionStatusException(false); // wrong error?
```

Comment:
👍 This is a good exception to use, it'll return with message "Site is not marked as adopted", which makes sense